interpreter/runtests.py

```diff
@@ -13,9 +13,10 @@
 inputDir = "test"
 expectDir = "expected-output"
 outputDir = os.path.join(inputDir, "output")
+isWindows = os.name == 'nt'
 
 parser = argparse.ArgumentParser()
-parser.add_argument('--wasm', metavar='<wasm-command>', default='./wasm')
+parser.add_argument('--wasm', metavar='<wasm-command>', default=('./wasm' if not isWindows else 'wasm.exe'))
```

Comment:
`os.path.join('.', 'wasm')` results in `./wasm` even on Windows which is not valid.
However, I found that `os.path.join(os.getcwd(), 'wasm')` is a valid command